pkg/kubelet/certificate/kubelet.go

```diff
@@ -61,7 +61,17 @@ func NewKubeletServerCertificateManager(kubeClient clientset.Interface, kubeCfg
 			StabilityLevel: compbasemetrics.ALPHA,
 		},
 	)
+	var certificateRenewFailure = compbasemetrics.NewGauge(
+		&compbasemetrics.GaugeOpts{
+			Namespace:      metrics.KubeletSubsystem,
+			Subsystem:      "certificate_manager",
+			Name:           "server_expiration_renew_failure",
```

Comment:
should this be `client_expiration_renew_failure`? do we want separate metrics for the client and serving cert renewal?

Reply:
In this instance, it should be the server, since we are in `NewKubeletServerCertificateManager`.